priv/app/function_browser.jsx

```diff
@@ -107,7 +105,7 @@ export default class FunctionBrowser extends React.Component {
   }
 
   matchFunSignature(input) {
-    var regex = /(\w+):(\w+)\/(\d+)/;
+    var regex = /([\w\.]+):(\w+)\/(\d+)/;
```

Comment:
True, that could be improved - an open question is if we want to care about other valid atom characters and combinations in module name? (e.g. `@` and others).